modules/planning/conf/scenario/park_and_go_config.pb.txt

```diff
@@ -0,0 +1,13 @@
+scenario_type: PARK_AND_GO
+park_and_go_config:
+{
+}
+
+stage_type: PARK_AND_GO_TRY
+stage_type: PARK_AND_GO_CRUISE
+stage_type: PARK_AND_GO_ADJUST
+
```

Comment:
switch stage #2 and #3. 
try (yes) -> cruise
try (fail) -> adjust -> cruise

and probably change "try" to "check"?